wger/gym/views/gym.py

```diff
@@ -111,15 +110,74 @@ def get_context_data(self, **kwargs):
         '''
         context = super(GymUserListView, self).get_context_data(**kwargs)
         context['gym'] = Gym.objects.get(pk=self.kwargs['pk'])
+        context['name'] = 'Active users'
         context['admin_count'] = len(context['object_list']['admins'])
         context['user_count'] = len(context['object_list']['members'])
-        context['user_table'] = {
-            'keys': [_('ID'),
-                     _('Username'),
-                     _('Name'),
-                     _('Last activity')],
-            'users': context['object_list']['members']
-        }
+
+        context['user_table'] = {'keys': [_('ID'), _('Username'), _('Name'), _('Last activity'), _('Status')],
+                                 'users': context['object_list']['members'],
+                                 "route": "active"}
+        return context
+
+
+class GymInactiveUserListView(LoginRequiredMixin, WgerMultiplePermissionRequiredMixin, ListView):
```

Comment:
i believe there could have been a better way to do this than creating a separate view. most of the code here is duplicated from `GymUserListView`